contents/newsletter/pivot-your-startup.md

```diff
@@ -0,0 +1,130 @@
+---
+title: Read this before you pivot your startup
+date: 2024-03-21
+author:
+  - james-temperton
+featuredImage: >-
+  https://res.cloudinary.com/dmukukwp6/image/upload/v1710126230/posthog.com/contents/images/blog/story-about-pivots.png
+featuredImageType: full
+tags:
+  - Startups
+crosspost:
+  - Product engineers
+  - Engineers
+  - Founders
+  - Blog
+---
+YouTube used to be a [dating website](https://www.theguardian.com/technology/2016/mar/16/youtube-past-video-dating-website). Twitter used to be a [podcasting platform](https://www.businessinsider.com/how-twitter-was-founded-2011-4). Wrigley, the chewing gum company, [originally sold soap](https://en.wikipedia.org/wiki/Wrigley_Company). Few startup founders get it right first time, and a well-timed and executed pivot is (almost) always worth it.
+
+Take Wrigley, for example. In the 1890s, when the company started [selling scouring soap](https://www.lib.uchicago.edu/collex/exhibits/sweet-home-chicago/history-chocolate-and-candy-making-chicago/wrigley-spearmen-work-and-play/), packs of chewing gum were an added extra to encourage people to order. But the gum soon proved more popular than the soap. The rest is chewy, minty history.
+
+At PostHog, we’ve pivoted more than most. It took us six pivots ([count ‘em](https://posthog.com/founders/story-about-pivots)) to land on product-market fit. Two years later, we’re speeding past 100,000 users.
+
+**This week’s theme is: How, and when, to pivot your startup.**
+
+> This article was first published in our newsletter, [Product for Engineers](https://newsletter.posthog.com/). It's all about helping engineers and founders build better products by learning product skills. We send it every two weeks. [Subscribe here](https://newsletter.posthog.com/subscribe).
```

Comment:
```suggestion
> This article was first published in our newsletter, [Product for Engineers](https://newsletter.posthog.com/). It's all about helping engineers and founders build better products by learning product skills. [Subscribe here](https://newsletter.posthog.com/subscribe).
```

Since it'll be changing soon-ish